src/modules/neo4j/README.md

```diff
@@ -1,38 +1,61 @@
 # Neo4j
 
-Neo4j is a highly scalable, robust native graph database.
+[Neo4j](https://neo4j.com/) is a highly scalable, robust native graph database.
 Let's test it!
 
 ## Examples
 
-Start a neo4j container and create a person node
+Test with jest to create a person node in neo4j
 
 ```typescript
-import neo4j from "neo4j-driver";
-import { Neo4jContainer } from "testcontainers";
+import neo4j, { Driver } from 'neo4j-driver'
+import { Neo4jContainer, StartedNeo4jContainer } from 'testcontainers'
 
-const container = await new Neo4jContainer().start();
+let container: StartedNeo4jContainer
+let driver: Driver
+let session: Session
 
-const driver = neo4j.driver(
-  container.getBoltUri(),
-  neo4j.auth.basic(container.getUsername(), container.getPassword())
-);
+beforeEach(async () => {
+  container = await new Neo4jContainer().withApoc().start()
+  driver = neo4j.driver(
+    container.getBoltUri(),
+    neo4j.auth.basic(container.getUsername(), container.getPassword())
+  )
+})
 
-const session = driver.session();
-const personName = "Chris";
+afterEach(async () => {
+  await session?.close()
+  await driver?.close()
+  await container?.stop()
+})
 
-try {
-  const result = await session.run("CREATE (a:Person {name: $name}) RETURN a", { name: personName });
+describe('neo4j', () => {
```

Comment:
is it intentional that the beforeEach/afterEach is outside of the describe here but not in the arango-db example?

Reply:
Same result when there is only one test suite. I'll move it in the global scope. Usually I would create a new test file when I've to switch a container setup.